td/tracking/templates/tracking/event_form.html

```diff
@@ -53,5 +185,84 @@ <h2 class="panel-title">New Translation Workshop/Event</h2>
 
 {% endblock body%}
 
-{% block extra_script %}
-{% endblock extra_script %}
\ No newline at end of file
+{% block custom_bottom_script %}
```

Comment:
All the js is built running npm run build. Put it into static/src/charter.js and then add require('./charter.js') to main.js. 

Sent from my iPad

> On Oct 5, 2015, at 3:31 PM, Vicky Leong <notifications@github.com> wrote:
> 
> In td/tracking/templates/tracking/event_form.html:
> 
> > \ No newline at end of file
> > +{% block custom_bottom_script %}
> Is this gonna be my own main.js that's imported into my event_form.html? Can I maybe just link a .js file to the HTML?
> 
> —
> Reply to this email directly or view it on GitHub.
>